added new nodes and updated the function

PRANJAL2208 opened the pull request.

PRANJAL2208 committed:
added new nodes and updated the function

zakmii commented on agents.py:
```diff
@@ -17,42 +17,42 @@
 
 Biological Process-related Relationships
-BIOLOGICALPROCESS_CHEMICALENTITY: Between Biological Process and Chemical
+BIOLOGICALPROCESS_CHEMICALENTITY: Between Biological Process and Chemical Entity
```
nit: ChemicalEntity

zakmii commented on agents.py:
```diff
@@ -17,42 +17,42 @@
+CHEMICALENTITY_DISEASE: Between Chemical Entity and Disease
+CHEMICALENTITY_CHEMICALENTITY: Between Chemical Entity and Chemical Entity
+CHEMICALENTITY_GENE: Between Chemical Entity and Gene
+CHEMICALENTITY_PROTEIN: Between Chemical Entity and Protein
```
nit: ChemicalEntity

We need to be very strict with the label

zakmii commented on agents.py:
```diff
@@ -90,29 +90,33 @@ def __init__(self, *args, **kwargs):
+PMID_CHEMICALENTITY: Between PMID and Chemical Entity
+PMID_DISEASE: Between PMID and Disease
+PMID_PROTEIN: Between PMID and Protein
+PMID_CELLULARCOMPONENT: Between PMID and Cellular Component
```
nit: CellularComponent

zakmii reviewed (changes requested):
please resolve label names

PRANJAL2208 committed:
Added new Species relation

Removed spaces in between BiologicalProcess, ChemicalEntity, MolecularFunction and CellularComponent.
Added a new species relation

zakmii committed:
Update agents.py
